src/server/config.rs

```diff
@@ -151,6 +156,12 @@ impl Config {
                 self.advertise_addr
             ));
         }
+        if !self.http_addr.is_empty() {
+            box_try!(config::check_addr(&self.http_addr));
+        }
+        if self.http_addr.starts_with("0.") {
```

Comment:
Why is it invalid when starts with 0? `0.0.0.0` seems to be a perfect binding address.